src/transaction/log_record.hpp

```diff
@@ -236,6 +238,15 @@ struct log_rec_donetime
   INT64 at_time;		/* Database creation time. For safety reasons */
 };
 
+typedef struct log_rec_group_commit LOG_REC_GROUP_COMMIT;
+struct log_rec_group_commit
+{
+  INT64 at_time;		/* time recorded by active server */
+  INT64 stream_pos;
+
+  int group_sz;
```

Comment:
```suggestion
  size_t redo_size;
```
It might be more flexible. We currently need to add two fields for committed without postpone and aborted transactions and three fields for those committed with postpone.
Pack/unpack can handle saving/extracting required information. redo_size can be used by recovery to know the total size of packed information.